src/keymap/mod.rs

```diff
@@ -18,6 +18,9 @@ pub trait KeyMap<'a, W: Write, T>: From<T> {
                 try!(self.editor().handle_newline());
                 done = true;
             }
+            Key::Ctrl('f') => {
```

Comment:
It turns out I cannot simply add Key::Right here, because it breaks test suites for vi and emaics mode.